.github/workflows/release.yml

```diff
@@ -31,7 +31,7 @@ on:
 
 jobs:
   release:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-latest-16-cores
```

Comment:
is this something Github supports natively or is this our custom runner configured somewhere?

Reply:
This is a custom runner that FINOS have set up. Only org admins have access to its configuration.

Reply:
@ivan-kyosev-gs this didn't seem to work though, is it because the runners are not yet available?

Reply:
@akphi I've added you to the discussion with FINOS. Looks like some of the settings for the runner are misconfigured at the moment.